spec/models/conference_spec.rb

```diff
@@ -778,6 +778,33 @@
     end
   end
 
+  describe '#scheduled_event_distribution' do
+    let!(:conference) { create(:conference) }
```

Comment:
@siddhantbajaj I understodd what you mean, but you don't need to call it in every test. If you know change `let!` by `let` it will work without changing  anything else.

As it is said in the [link](https://www.relishapp.com/rspec/rspec-core/v/2-5/docs/helper-methods/let-and-let), `let` is using lazy evaluation, and the variable will be created as soon as it is called the first time. That means that if you create another test that don't use it it won't be created. In your first test if would be created just before creating ` confirmed_unscheduled_event`. In the second would be created in the expect as it is called there.

Do you understand what I mean? :slightly_smiling_face: